.github/workflows/version-canary.yml

```
@@ -0,0 +1,61 @@
+name: rsnd
```

Let's start give a more concrete names for the workflows.
```suggestion
name: rsnd
```

---

The problem is that it'd get repetitive if we used something like `release/version` or `release/release`. What names did you have in mind?